src/webgpu/shader/execution/expression/expression.ts

```diff
@@ -912,8 +976,6 @@ ${wgslHeader(parameterTypes, resultType)}
 
 ${wgslOutputs(resultType, cases.length)}
 
-${wgslValuesArray(parameterTypes, resultType, cases, expressionBuilder)}
```

Comment:
Realized this wasn't needed for AF, since the inputs are inlined into the shader.